articles/112_generated_interfaces_cpp.md

```diff
@@ -104,6 +104,13 @@ All other constants are declared as `static const` members in the struct and the
 ### Constructors
 
 The *default constructor* initializes all members with their default value.
+In some cases this may not be desirable, since these fields will often be immediately overwritten with user-provided values.
+Therefore, the constructor takes an optional directive of type `enum rcl_msg_init_type` to control how initialization is done:
```

Comment:
Since the symbol should be independent of the ROS client library I don't think `rcl` is a good prefix. Maybe `rosidl_runtime_cpp`? `rosidl_cpp_runtime` sounds nicer but all the generator packages are named `rosidl_generator_*`...